languages/csharp/exercises/.docs/cli.md

```diff
@@ -4,8 +4,8 @@ You can run the tests by opening a command prompt in the exercise's directory, a
 
 Initially, only the first test will be enabled. This is to encourage you to solve the exercise one step at a time. Once you get the first test passing, remove the `Skip` property from the next test and work on getting that test passing.
 
-Once none of the tests are skipped and they are all passing, you can submit your solution by opening a command prompt in the exercise's directory and running the
-using [`exercism submit` command][docs-exercism-cli].
+Once none of the tests are skipped and they are all passing, you can submit your solution by opening a command prompt in the exercise's directory and 
+running the [`exercism submit` command][docs-exercism-cli].
```

Comment:
The linebreak seems a little arbitrary, doesn't it? How about merging these two lines?

The linebreak even seems to have been the cause of the incongruence.

Reply:
It occured to me that it'd be faster if someone just did that, but apparently I can't:
```
To github.com:tehsphinx/v3.git
 ! [remote rejected] patch-2 -> patch-2 (permission denied)
error: failed to push some refs to 'git@github.com:tehsphinx/v3.git'
```
Any idea why I can't push to this feature branch? I thought being able to merge pull requests gave me the privilege to push to the involved feature branches? Is this because @tehsphinx didn't check the mark that allows this when submitting the pull request?

Reply:
I would also have expected that. I'm not well-versed in GitHub permissions. Let me see if I am allowed to do this.

Reply:
Hmmm, if I do a `git fetch` I don't see the `tehsphinx-patch-2` branch.

Reply:
The feature branch is in his fork. This is the branch I cannot push to.